Modules/Core/Common/include/itkResourceProbe.hxx

```diff
@@ -558,16 +558,6 @@ ResourceProbe<ValueType, MeanType>::PrintJSONSystemInformation(std::ostream & os
   os << "  }";
 }
 
-
-// This protected member function that was introduced with ITK 4.8 has been deprecated
-// as of ITK 5.0. Please do not call or override this member function.
```

Comment:
I think the policy is to remove deprecated things at the beginning of a new version. When starting 5.0 development, @hjmjohnson removed ITKv3 compatibility and everything else which was declared deprecated during 4.x. We declared this deprecated as of 5.0, so removing it in 5.2 already seems a bit fast.

Reply:
> We declared this deprecated as of 5.0, so removing it in 5.2 already seems a bit fast.

Well, it was deprecated more than two years ago. In another decade. It feels like a long time... long before the corona pandemic...  😢 

Technically, I would prefer to prevent any user from calling `GetSystemInformation()` (as it does nothing useful), as well as from overriding the function (as it is not being called by internally by `ResourceProbe` anymore). The sooner the better. But if you think otherwise, I'll leave it up to you.

Reply:
@N-Dekker   I think we should start pushing for ITKv6.0 that only supports C++14 (or C++17), cmake 3.16+, python 3.6+ and make another round of deprecations.

:)